http/error.go

```diff
@@ -18,20 +13,12 @@ func GetErrorMessage(err error) string {
 		return ""
 	}
 
-	if richErr, ok := err.(errors.Error); ok {
-		if msg, ok := errorMessages[richErr.Message()]; ok {
-			return msg
-		}
-	} else {
-		if msg, ok := errorMessages[err.Error()]; ok {
-			return msg
-		}
-	}
-
-	err = errors.Unwrap(err)
-	if err == nil {
-		return ""
+	switch true {
+	case errors.Is(err, ErrDuplicatedWalletAddress):
+		return "Address already registered, please generate a new address"
```

Comment:
```suggestion
		return "Wallet already registered for your endpoint or another endpoint"
```
It's hard to come up with a good error message for this situation, because it could be that the wallet is registered for the supplied endpoint or another endpoint, so we can't say "please do this" or "please do that" because we don't know scenario it is. Should we split up into two different error messages?

Reply:
I would use `Wallet already registered for an endpoint`.